scanpipe/pipes/docker.py

```diff
@@ -61,23 +61,23 @@ def extract_layers_from_images(project, images):
     Return the `errors` that may have happen during the extraction.
     """
     errors = []
-
+    # FIXME: use container-inspector extract instead
```

Comment:
Could you provide some context and reason about this change?

Reply:
@JonoYang Could you enter a new issue with some details of the problem? This would have more chance to be worked on than a comment in the code.
Thanks!

Reply:
@tdruez there are some issues that @JonoYang uncovered in how code is being extracted AFAIRC.

Reply:
@pombredanne I would prefer a ticket nonetheless as those `# FIXME` tend to never get dealt with.

Reply:
Good point, https://github.com/nexB/scancode.io/issues/251 added